cargo-shuttle/src/client.rs

```diff
@@ -92,48 +92,12 @@ impl Client {
         self.get(path).await
     }
 
-    pub async fn create_project(
-        &self,
-        project: &ProjectName,
-        config: project::Config,
-    ) -> Result<project::Response> {
-        let path = format!("/projects/{}", project.as_str());
-
-        self.post(path, Some(config))
-            .await
-            .context("failed to make create project request")?
-            .to_json()
-            .await
-    }
-
-    pub async fn clean_project(&self, project: &ProjectName) -> Result<Vec<String>> {
-        let path = format!("/projects/{}/clean", project.as_str(),);
-
-        self.post(path, Option::<String>::None)
-            .await
-            .context("failed to get clean output")?
-            .to_json()
-            .await
-    }
-
-    pub async fn get_project(&self, project: &ProjectName) -> Result<project::Response> {
```

Comment:
I did not remove this, but I am not sure how much use it would have over getting a deployment/service status.